app/models.py

```diff
@@ -166,14 +168,49 @@ class ChangePasswordPayload(BaseModel):
     password: str
     newPassword: str
 
+
 class ForgotPasswordPayload(BaseModel):
     token: str
     newPassword: str
 
+
 class JoinEventPayload(BaseModel):
     food: Optional[bool]
     transportation: Optional[bool]
     dietaryRestrictions: Optional[str]
 
+
 class PenaltyInput(BaseModel):
-    penalty: int=Field(ge=0, description="Penalty must be larger or equal to 0")
+    penalty: int = Field(
+        ge=0, description="Penalty must be larger or equal to 0")
+
+
+class JobItem(BaseModel):
+    id: UUID4
+    company: str
+    title: str
+    type: str
+    img: str
+    tags: List[str]
+    description_preview: str
+    description: str
+    start_date: datetime
+    published_date: datetime
+    location: str
+    link: str
+    due_date: datetime
+
+
+class JobItemPayload(BaseModel):
+    company: str
+    title: str
+    type: str
+    img: str
+    tags: List[str]
+    description_preview: str
+    description: str
+    start_date: datetime
+    published_date: datetime
+    location: str
+    link: str
+    due_date: datetime
```

Comment:
An idea here if here could be:
```
 Class JobItemPayload(BaseModel):
    # all fields
  Class JobItem(JobItemPayload):
    id: UUID4
```
Meaning JobItem always extend JobItemPayload

Reply:
Hahah don't know why i did this :smile: